### PR TITLE
Inline doc for DataFrame.assign()

### PR DESCRIPTION
**Pull request by shril**
Hi @HyukjinKwon,

This PR has inline documentation for Koalas assign function.
This is the link to pandas assign function doc [pandas.DataFrame.assign](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.assign.html).

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
@shril, btw, mind if I ask to leave the link you referred for the doc in PR description :-)? that should make the review faster

**Comment by shril** [comment]
> @shril, btw, mind if I ask to leave the (Pandas doc) link you referred for the doc in PR description :-)? that should make the review faster

Sure thing @HyukjinKwon.

**Comment by rxin** [comment]
@shril can you take the pipe one out? There's another PR for it https://github.com/databricks/koalas/pull/262 that looks like it's good to go.

**Comment by shril** [comment]
@rxin I am removing it. I do feel that my examples were better at explaining.
If you still feel like it. I will remove it straightaway.

> @shril can you take the pipe one out? There's another PR for it #262 that looks like it's good to go.

**Comment by shril** [comment]
> @shril can you take the pipe one out? There's another PR for it #262 that looks like it's good to go.

Removed.

**Comment by rxin** [comment]
@shril looks like the linter failed.

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -733,13 +733,6 @@ def to_koalas(self):
            __index_level_0__  col1  col2
         0                  0     1     3
         1                  1     2     4
-
-        Calling to_koalas on a Koalas DataFrame simply returns itself.
-
-        >>> df.to_koalas()
```
Comment (rxin) [inline databricks/koalas/frame.py]: why was this removed?

**Comment by rxin** [comment]
@shril looks like the diff is messed up right now with a lot of other changes.

**Comment by shril** [comment]
> @shril looks like the diff is messed up right now with a lot of other changes.

I am shifting it to a new branch.

**Comment by rxin** [comment]
Closing this one in favor of https://github.com/databricks/koalas/pull/267
